force-app/main/default/lwc/assetManagementRelated/__tests__/assetManagementRelated.test.js

```diff
@@ -0,0 +1,204 @@
+import getAssetsByAccount from '@salesforce/apex/AssetManagementController.getAssetsByAccount';
+import renewCancel from '@salesforce/apex/AssetManagementController.renewCancelAsset';
+
+const getMockAssets = require('./data/getAssetList.json');
+const selectedRows = [ {
+  "Id" : "02iRO0000005mtQYAQ",
+  "Name" : "Revenue Cloud Test Product 1",
+  "Product2Id" : "01tRO000000KQYvYAO",
+  "AccountId" : "001RO000003T1GUYA0",
+  "LifecycleStartDate" : "2022-01-01T00:00:00.000+0000",
+  "LifecycleEndDate" : "2022-12-31T23:59:59.000+0000",
+  "RenewalTerm" : 1.00,
+  "RenewalTermUnit" : "Months",
+  "Product2" : {
+    "Id" : "01tRO000000KQYvYAO",
+    "Name" : "Revenue Cloud Test Product 1"
+  }
+}];
+
+const APEX_SUCCESS =
+    '';
+
+
+jest.mock(
+  '@salesforce/apex/AssetManagementController.getAssetsByAccount',
+  () => {
+      const {
+          createApexTestWireAdapter
+      } = require('@salesforce/sfdx-lwc-jest');
+      return {
+          default: createApexTestWireAdapter(jest.fn())
+      };
+  },
+  { virtual: true }
+);
+
+jest.mock(
+  '@salesforce/apex/AssetManagementController.renewCancelAsset',
+  () => {
+      const {
+          createApexTestWireAdapter
+      } = require('@salesforce/sfdx-lwc-jest');
+      return {
+          default: createApexTestWireAdapter(jest.fn())
+      };
+  },
+  { virtual: true }
+);
+
+
+describe('c-asset-management-related', () => {
+  afterEach(() => {
+      // The jsdom instance is shared across test cases in a single file so reset the DOM
+      while (document.body.firstChild) {
+          document.body.removeChild(document.body.firstChild);
+      }
+      // Prevent data saved on mocks from leaking between tests
+      jest.clearAllMocks();
+  });
+
+  //Helper method to wait until the microtask queue is empty.
+  async function flushPromises() {
+      return Promise.resolve();
+  }
+
+  it('renders two rows in the lightning datatable', async () => {
+      const element = createElement('c-asset-management-related', {
+          is: AssetManagementRelated
+      });
+      document.body.appendChild(element);
+
+      // Emit data from @wire
+      getAssetsByAccount.emit(getMockAssets);
+
+      // Wait for any asynchronous DOM updates
+      await flushPromises();
+
+      const tableEl = element.shadowRoot.querySelector('lightning-datatable');
+
+      //Validate the datatable is populated with correct number of records
+      expect(tableEl.data.length).toBe(getMockAssets.length);
+  });
+
+  it('Renew button is disabled', async () => {
+
+      // Create initial element
+      const element = createElement('c-asset-management-related', {
+        is: AssetManagementRelated
+      });
+      document.body.appendChild(element);
+
+      // Emit data from @wire
+      getAssetsByAccount.emit(getMockAssets);
+      
+      // Wait for any asynchronous DOM updates
+      await flushPromises();
+
+      //Update multiple records with the INPUT_PARAMETERS and simulate the Save event
```

Comment:
This comment seem inaccurate? You're just getting the button element from the DOM.